docs/resources/loadbalancer_credential.md

```diff
@@ -4,12 +4,15 @@ page_title: "stackit_loadbalancer_credential Resource - stackit"
 subcategory: ""
 description: |-
   Load balancer credential resource schema. Must have a region specified in the provider configuration.
+  !> The stackit_loadbalancer_credential resource has been deprecated and will be removed after November 13th 2024. Please use stackit_loadbalancer_observability_credential instead, which offers the exact same functionality.
```

Comment:
This notation is similar to https://github.com/stackitcloud/terraform-provider-stackit/blob/6eb1310056f1e3c1758bdbc860485d47a7764199/docs/resources/postgresql_credential.md?plain=1#L14, and will appear [like this](https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs/resources/postgresql_credential) in the registry

Reply:
Thank you for the comment explaining this!